job_runner/reporting/event.py

```diff
@@ -79,7 +81,7 @@ def _initialize_state_machine(self):
 
     def report(self, msg):
         new_state = None
-        for state in self._states.keys():
```

Comment:
Does it not make sense to use keys() when we are just discarding the values anyway?

Reply:
It does, distracted by the pylint message which says `items()`..